ContainerController/ContainerController.xcodeproj/project.pbxproj

```diff
@@ -113,7 +113,6 @@
 				87114B141D55EBA600C58D85 /* Frameworks */,
 				87114B151D55EBA600C58D85 /* Headers */,
 				87114B161D55EBA600C58D85 /* Resources */,
-				87114B5B1D55EE7C00C58D85 /* Run Swiftlint */,
```

Comment:
SwiftLint should still be used. There is also an example project workspace. This one should compile always, currently it doesn't as SwiftLint errors prevent it. Feel free to update to SwiftLint 0.30.1 and remove the underscore prefixes to fix the SwiftLint errors. Running SwiftLint in the framework target should stay as well